### PR TITLE
Modifies license copyright.

### PR DESCRIPTION
**Pull request by barath**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 BESS is distributed under the following BSD3 license:
 
-Copyright (c) 2014-2016, Regents of the University of California and Nefeli Networks, Inc.  All rights reserved.
+Copyright (c) 2014-2016, Regents of the University of California.  All rights reserved.
```
Comment (melvinw) [inline LICENSE]: There's an extra space between "." and "All"

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+Copyright (c) 2016, Nefeli Networks, Inc.  All rights reserved.
```
Comment (melvinw) [inline LICENSE]: Same here

File: LICENSE (hunk 1 above)
Comment (barath) [inline LICENSE]: Will update.  My habit has always been two spaces after periods, but I know that's less common these days:

https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing

**Comment by barath** [comment]
Got feedback from Kevin, made an additional edit.

**Comment by melvinw** [comment]
Thanks! 
moneycat

**Pull request merged**